ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/HttpHeadersMap.kt

```diff
@@ -29,17 +30,17 @@ private const val HEADER_SIZE = 8
 private const val HEADER_ARRAY_POOL_SIZE = 1000
 
 @ThreadLocal
-private val EMPTY_INT_ARRAY = IntArray(0)
+private val EMPTY_INT_LIST = mutableListOf<Int>()
```

Comment:
should we use `emptyList()` from stdlib instead?

Reply:
The instance of `emptyList()` is `List`, but we need `MutableList` here

Reply:
Should we create a new list every time? I'm afraid it can be accidentally freezed

Reply:
It's thread local and should not be modified. If it is modified, it will crash with `IndexOutOfBoundException`, since there is no adding elements to it, only setting